rules/windows/process_creation/proc_creation_win_attrib_system_susp_paths.yml

```diff
@@ -0,0 +1,40 @@
+title: Set Suspicious Files as System Files Using Attrib
+id: efec536f-72e8-4656-8960-5e85d091345b
+related:
+    - id: bb19e94c-59ae-4c15-8c12-c563d23fe52b
+      type: derived
+status: experimental
+description: Detects usage of attrib with "+s" option to set suspcious files as system files
```

Comment:
proc_creation_win_attrib_system.yml is a gerneric "attrib +s".
"
Detects usage of attrib with "+s" option to set suspcious script or executable as system files to hide then.
Rule Check (is limited to ?) uncommun folder for system file.
"

Reply:
The "proc_creation_win_attrib_system.yml" (bb19e94c-59ae-4c15-8c12-c563d23fe52b) Is generic and has a level of "low". This one focuses on special cases to increase the severity of the rule.

Reply:
It is not to change the rule but help a new SOC N1 why he get a high alert.
Just say `Detects usage of attrib with "+s" option to set suspcious files as system files` seen a lot short for description for me.

Reply:
Ah, I thought you saw something wrong in the rule. Sorry for the misunderstanding. Okay, I will update the description.